cmd/kubeadm/app/util/system/kernel_validator.go

```diff
@@ -78,7 +78,7 @@ func (k *KernelValidator) Validate(spec SysSpec) (error, error) {
 
 // validateKernelVersion validates the kernel version.
 func (k *KernelValidator) validateKernelVersion(kSpec KernelSpec) error {
-	glog.Infof("Validating kernel version")
+	fmt.Println("[preflight] Validating kernel version")
```

Comment:
we shouldn't imply pre-flight checks in the validator code.

we should do this instead:
`glog.V(1).Infof(...`

for both.